notifier.rst

```diff
@@ -4,11 +4,6 @@
 Creating and Sending Notifications
 ==================================
 
-.. versionadded:: 5.0
```

Comment:
We should only remove the experimental but keep the introduction to Symfony. Same for the other

Reply:
no strong opinion about it, but we don't do it for other components.

ie. String component: https://github.com/symfony/symfony-docs/commit/c86a35d21b778d0c0543f899ba610f0b67735e4f

in Mailer, we replaced the sentence as you suggest: https://github.com/symfony/symfony-docs/commit/6a9dc1b38e995e7c68aa30c173ac2545ab95123c, but then you removed the block in https://github.com/symfony/symfony-docs/commit/f990c8188b523233063c96a8040ea39de1854a5c

Reply:
Yes because the mailer was introduced in version 4 so we don't show it in 5 and above anymore, but notifier is introduced in 5 which needs to exist and will be removed in docs version 6

Reply:
fixed